tests/test.nix

```diff
@@ -85,13 +85,16 @@ let
       };
     };
   };
-  tsp-create = pkgs.writeScript "create" ((pkgs.callPackage ../. {}).create disko-config);
-  tsp-mount = pkgs.writeScript "mount" ((pkgs.callPackage ../. {}).mount disko-config);
-in makeTest' {
-  name = "disko";
-
-  nodes.machine =
-    { config, pkgs, modulesPath, ... }:
+  tsp-create = pkgs.writeScript "create" ''
+    #!${pkgs.bash}/bin/bash
```

Comment:
```suggestion
  tsp-create = pkgs.writers.writeDashBin "create" ''
```

Reply:
https://github.com/nix-community/disko/pull/18 will require bash features for sure and I don't see the value of adding dash give that its not in the nixos closure and makes implementing features like this harder.

Reply:
The point is that systemd will refuse to execute a script without a shebang.

Probably the shebang should be added in disko directly, but that would require disko to get `pkgs` as an input. In turn I think that should be a requirement to make it really predictable, so it always uses pinned packages in the scripts. See https://github.com/nix-community/disko/pull/20#issuecomment-1225626395.

I just didn't want to touch the lib in this PR, to make sure I was testing a real problem in current disko.